CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+# Contributing to [Project-Name]
+

```

Comment:
what python version are we using ?

Reply:
Is 3.12.3 ok for you? I've just added info about it to CONTRIBUTING.md file